SPECS/kernel-azure/config

```diff
@@ -1797,8 +1797,6 @@ CONFIG_ETHTOOL_NETLINK=y
 #
 # Device Drivers
 #
-CONFIG_HAVE_EISA=y
-# CONFIG_EISA is not set
```

Comment:
result of : https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/3914a222d6733c8c4b37c0c57c5bbb4b4fc989a2